### PR TITLE
Documentation of humann2.jl functions

### PR DESCRIPTION
**Pull request by annelle-abatoni**

### CODE REVIEW

File: Tutorial.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+```julia
+# EXAMPLES WITH KEEPUNIDENTIFIED = TRUE AND FALSE
+```
+
+### Indexing CommunityProfiles
+CommunityProfiles can be indexed using items in "Thing names" and "Place names".
+
+```julia
+julia> profile = metaphlan_profiles("test/files/metaphlan_multi_test.tsv")CommunityProfile{Float64, Taxon, MicrobiomeSample} with 42 things in 7 places
```
Comment (kescobo) [inline Tutorial.md:101]: looks like a newline is missing here

File: Tutorial.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+# Welcome to the BiobakeryUtils tutorial
+
+## Using BiobakeryUtils with [MetaPhlAn](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-3.0) files
+
+### Input file format
+
+BiobakeryUtils accepts `.tsv` files outputted from MetaPhlAn.
+Files can either contain a merged table or one or multiple single table(s).
```
Comment (kescobo) [inline Tutorial.md:8]: Maybe mention that this comes from the `metaphlan_merge_tables` command, and link to the relevant MetaPhlAn docs?

File: Tutorial.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+When creating a CommunityProfile from a merged table, data from unidentified taxa can be kept in by setting the `keepunidentified` flag to `true`. The default is to filter out unidentified data.
+
+```julia
+# EXAMPLES WITH KEEPUNIDENTIFIED = TRUE AND FALSE
+```
+
+### Indexing CommunityProfiles
+CommunityProfiles can be indexed using items in "Thing names" and "Place names".
```
Comment (kescobo) [inline Tutorial.md:98]: We should really fix this to be "feature names" and "sample names" - it [requires this](https://github.com/EcoJulia/EcoBase.jl/pull/17), I'll get a release of ecobase made

File: src/humann2.jl [hunk 4]
```diff
@@ -35,7 +35,13 @@ function humann2_rename(df::DataFrame; kind::String="ec")
     return new_df[!,1]
 end
 
+"""
+    humann2_barplots(df::DataFrame, metadata::AbstractArray{<:AbstractString,1}, outpath::String)
 
+Wrapper for `humann2_barplots` script,
+sorts and filters through a humann2 DataFrame, matches it with its respective metadata,
+and plots them together.
+"""
```
Comment (kescobo) [inline src/humann2.jl:44]: I'm not certain this actually works anymore. Might be worth testing

**Review by kescobo — CHANGES_REQUESTED** [review]
This is looking pretty good! I think one improvement would be to make it more closely track the [metaphlan tutorial](https://github.com/biobakery/biobakery/wiki/metaphlan3 )itself, possibly even using files from there. For example, there are a couple of places where command line tools are used to visualize things, we could show how to do that with this package.

But for now, OK to just fix the few things I commented on, then open issues for the other ones to deal with in a sepate PR

File: Tutorial.md (hunk 3 above)
Comment (kescobo) [reply Tutorial.md:98]: https://github.com/BioJulia/Microbiome.jl/pull/70

**Comment by kescobo** [comment]
This has some substantial overlap with #50 now, and is mostly not about the `humann` functions themselves. @anikaluo and @annelle-abatoni, can you work together to sort out what parts of this can be combined with #50 or vice versa, and let me know?

File: Tutorial.md (hunk 2 above)
Comment (annelle-abatoni) [reply Tutorial.md:8]: Is this the one you mean: https://github.com/biobakery/MetaPhlAn/blob/master/metaphlan/utils/merge_metaphlan_tables.py ?

**Pull request closed without merging**